Testing/Exec/ReactEval/inputs.3d_1dArray

```diff
@@ -5,8 +5,8 @@ fuel_name = CH4
 reactFormat = 1dArray
 chem_integrator = "ReactorRK64"
 
-ode.dt  = 1.e-05
-ode.ndt = 1
+ode.dt  = 2.e-05
```

Comment:
Are the changes in this file and in the other ReacEval input file deliberate? Do we want a ReactorBDF input file too?

Reply:
yes, without these, even cvode was failing to integrate this case